Cubical/Categories/NaturalTransformation/Properties.agda

```diff
@@ -0,0 +1,145 @@
+
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+module Cubical.Categories.NaturalTransformation.Properties where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism renaming (iso to iIso)
+open import Cubical.Data.Sigma
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor
+open import Cubical.Categories.Morphism renaming (isIso to isIsoC)
+open import Cubical.Categories.NaturalTransformation.Base
+
+private
+  variable
+    ℓC ℓC' ℓD ℓD' : Level
+
+open isIsoC
+open NatIso
+open NatTrans
+open Precategory
+open Functor
+
+module _ {C : Precategory ℓC ℓC'} {D : Precategory ℓD ℓD'} where
+  private
+    _⋆ᴰ_ : ∀ {x y z} (f : D [ x , y ]) (g : D [ y , z ]) → D [ x , z ]
+    f ⋆ᴰ g = f ⋆⟨ D ⟩ g
+
+  -- natural isomorphism is symmetric
+  symNatIso : ∀ {F G : Functor C D}
+            → F ≅ᶜ G
+            → G ≅ᶜ F
+  symNatIso η@record { trans = record { N-ob = N-ob ; N-hom = N-hom }
```

Comment:
Wouldn't this look nicer as a definition by copatterns?

Reply:
Yeah I agree. I think this was from before I really new how to work with copatterns.

Reply:
Can you fix it please?

Reply:
I fixed it in my push last week, but I think it didn't update because the Travis build failed. I'm not sure how to fix the build error. It says there's an error with `Cubical.ZHomology.GroupStructure`, but when I check it on my computer works. Do you know how to get it to work?

Reply:
That's probably the error that was fixed by #508, try merging from master.

Reply:
Ah okay, just pushed, hopefully it checks out